templates/package.json

```diff
@@ -6,7 +6,11 @@
     "scripts": {
         "check-sensitive": "npx eslint --plugin 'no-secrets' --cache --ignore-pattern 'package.json' --ignore-pattern 'package-lock.json'",
         "precommit": "npm run check-sensitive && lint-staged",
-        "version": "../.github/scripts/pkg-version.sh core-template"
+        "version": "bash ../.github/scripts/pkg-version.sh template-sync && ../.github/scripts/pkg-version.sh core-template"
```

Comment:
what is  core-template for? didn't find the option in pkg-version.sh

Reply:
the core-template do sync up config to download templates version in fx-core.
this triggered once templates version change.